src/main/webapp/synchronizer.js

```diff
@@ -13,11 +13,14 @@
 // limitations under the License.
 
 import {seekDiscussion} from './view/discussion/discussion.js';
-import {seekTranscript} from './view/transcript/transcript.js';
+import TranscriptSeeker from './view/transcript/transcript-seeker.js';
 
 let lastTime;
 
 const TIME_INTERVAL_MS = 100;
+// TODO: Retrieve the transcriptSeeker from the TranscriptArea instead
+// once #255 is merged into master.
+const transcriptSeeker = new TranscriptSeeker('event controller');
```

Comment:
Can you add a TODO to remove transcript seeker from this class once the eventListeners are added?

Reply:
Or is this staying?

Reply:
No, that is just there temporarily. I added a TODO to move it out.